package.json

```diff
@@ -154,6 +173,23 @@
           "command": "license.chooseMultipleLicenses",
           "when": "workspaceFolderCount > 0 && !virtualWorkspace"
         }
+      ],
+      "explorer/context": [
+        {
+          "command": "license.chooseLicense",
+          "group": "1_modification",
```

Comment:
As I can see in [docs](https://code.visualstudio.com/api/references/contribution-points#Sorting-of-groups), `1_modification` is in `editor/context`, not in `explorer/context`, maybe it should be `2_workspace` - Commands related to workspace manipulation?

Correct me if I wrong.

Reply:
My intention is to add a separate group. If you set like this, you can see the effect in the picture.

Let me know what you think.
```
"explorer/context": [
        {
          "command": "license.chooseLicense",
          "group": "0_modification",
          "when": "explorerResourceIsFolder"
        },
        {
          "command": "license.addDefaultLicense",
          "group": "not exist",
          "when": "explorerResourceIsFolder"
        },
        {
          "command": "license.chooseMultipleLicenses",
          "group": "2_workspace",
          "when": "explorerResourceIsFolder"
        }
      ]
```
![image](https://github.com/ultram4rine/vscode-choosealicense/assets/22630678/85b3be55-299c-4d42-bd05-73e5b0328983)